resources/js/src/app/services/UrlService.js

```diff
@@ -26,11 +26,12 @@ export function getUrlParams(urlParams)
 
 export function setUrlParams(urlParams)
 {
-    var pathName = window.location.pathname;
+    var pathName = store.state.navigation.currentCategory !== null ? store.state.navigation.currentCategory.url : window.location.pathname;
```

Comment:
... and use it here like this `var pathName = isDefined(store.state.navigation.currentCategory) ? store.state.navigation.currentCategory.url : window.location.pathname;`

Reply:
Done